scripts/assets/maven_dependencies.textproto

```diff
@@ -830,6 +928,16 @@ maven_dependency {
     }
   }
 }
+maven_dependency {
+  artifact_name: "org.codehaus.mojo:animal-sniffer-annotations:1.18"
+  artifact_version: "1.18"
+  license {
+    license_name: "The Apache Software License, Version 2.0"
```

Comment:
It could have been an error on my part. I deleted it and re-ran the script, found that it required manual update. 
I checked the other licenses and they are all correct.